packages/developer-portal/package.json

```diff
@@ -73,6 +75,7 @@
     "serverless-s3-deploy": "^0.10.1",
     "serverless-s3-remover": "^0.6.0",
     "serverless-single-page-app-plugin": "^1.0.4",
+    "supertest-graphql": "^1.1.4",
```

Comment:
Shouldn't this be in the GQL package not Dev Portal?

Reply:
yes it should